### ISSUE TITLE
Circlator using a gene evicted from Uniprot 1 year ago

### ISSUE THREAD

**Issue by tseemann** [sanger-pathogens/circlator#73]
This came up on one of our `circlator fixstart` runs on a K.pneumo chromosome:

`tr|U1EGA9|U1EGA9_ENTGA`

http://www.uniprot.org/uniprot/U1EGA9

```
This entry is obsolete
On September 16, 2015 this entry was deleted.
For previous versions of this entry, please look at its history.
```

(Found by @annasyme)

**Comment by martinghunt** [comment]
Sorry for slow reply - have been away. Thanks for spotting that. I'll update the default fasta that circlator uses. A quick fix for you in the meantime is to download your own using

    circlator get_dnaa out

I just double-checked and this indeed does not include U1EGA9.

**Comment by martinghunt** [comment]
Fixed in release 1.3.1
